packages/react-storage/src/components/StorageBrowser/Views/Controls/Paginate.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { StorageBrowserElements } from '../../context/elements';
+
+interface PaginateItem<T extends StorageBrowserElements>
+  extends Pick<T, 'ListItem' | 'Button'> {
+  (): React.JSX.Element;
+}
+
+export interface PaginateControl<T extends StorageBrowserElements> {
+  (): React.JSX.Element;
+  List: T['OrderedList'];
```

Comment:
Just checking my assumptions and some questions:
- List is a `<nav><ol></ol></nav>` compound component. Do we want to keep calling these wrapper elements Container since we do that a lot elsewhere? `Container: T['Nav'];` in this case.
- Current (and Next,Previous) are a `<li><button></button></li>` compound component?